### ISSUE TITLE
Road to gas miner abolishment

### ISSUE THREAD

**Issue by Peptide90** [space-wizards/space-station-14#8311]
## Description

Tl;Dr gas miners are little reward all risk. In the past people pipe them into distro and instantly ruin the round.

To fix this we have a two prong approach:
- Create a CO2 recycling device to return some of the spent O2 and a magic waste product (or non) at the cost of medium to high power draw.
- Add the magnetic gas storage device (SMES for gas) and have this purchased from cargo for a high price or found on salvage wrecks.

A proper atmos setup should have minimal gas loss anyway so using cargo should be a last resort. We can then relegate gas miners to rare use cases and late game tech. In future gas miners will also be very high power draw too so this adds alternatives.

**Comment by TheIntoxicatedCat** [comment]
"there is Sabatier reaction which takes in place 200–300 degree celcius, which is;

CO2+2(H2)→ O2+CH4

This reaction is also used in ISS for O2 source and by product of methane."
We'd need methane and hydrogen, or alternatively, plasma, I guess, gas and methane could be used for welding fuel or jetpacks, etc.

**Comment by Emisse** [comment]
we have tritium which is just H3, we could do some magic space jumbo

**Comment by Peptide90** [comment]
Yeah we bikeshedded this in discord and came up with just magic the hydrogen away. I think there’s little merit in taking up our 8th gas slot with it unless it’s also used for stuff like fusion and funny bombs but it would need a non sabotage use too.

I also jokingly said you can give a structure the lung component to turn co2 to oxygen for power but I’m sure maintainers will want something more robust than that.

**Comment by efzapa** [comment]
based beyond belief, fuck gas miners

**Comment by efzapa** [comment]
imo gas miners should be drawing gigawatts for what they pump out imo

**Comment by youarereadingthis** [comment]
Realistically I imagine that gas miners would put out randomly fluctuating amounts, especially if there are multiple of them mining in the same spot.
Sometimes they might put out a lot, and sometimes they may not put out enough.

Combined with powered recyclers I think that would be a good way of balancing them that isn't just a strict power requirement.

**Comment by ike709** [comment]
2CO₂ + 4T → O₂ + 2C + 2T₂O

This would be a theoretically viable modified Sabatier reaction that combines 2 carbon dioxide molecules with 4 tritium molecules to form oxygen, 2 carbon atoms, and 2 superheavy water molecules (which could be faked with radioactive water).

**Comment by Peptide90** [comment]
> Realistically I imagine that gas miners would put out randomly fluctuating amounts, especially if there are multiple of them mining in the same spot. Sometimes they might put out a lot, and sometimes they may not put out enough.
> 
> Combined with powered recyclers I think that would be a good way of balancing them that isn't just a strict power requirement.

We bikeshedded some more and came up with the following solutions:
- make gas miners output far less gas and cap their max pressure they work in to like 3500kpa
- Still do the bigger gas storage tanks that can be moved around as a supplement for ships etc.
- Possibly have pressure release valves that can be put on the network (constructed) to prevent over pressure.
- Gas recycler will work too, only issue with Ikes suggestion is Tritium is pretty rare at present and highly bomb able.

**Comment by Partmedia** [comment]
Proposed gas recycler in #9934.

**Comment by Peptide90** [comment]
Closing this unless maintainers / PMs decide getting rid of them is still a good idea.

Only thing missing IMO is a way to purchase larger volumes of air (magnetic storage units) or produce it from ice rocks. And or the red reserve gas tanks that can't be built or moved.
